lucille-core/src/main/java/com/kmwllc/lucille/stage/ParseFloats.java

```diff
@@ -23,8 +28,13 @@ public ParseFloats(Config config) {
   public Iterator<Document> processDocument(Document doc) throws StageException {
     if (doc.has(field)) {
       String value = doc.getString(this.field);
-      List<Float> floats = Arrays.stream(value.replaceAll("\\[", "").replaceAll("\\]", "").replaceAll(",", "").split(" "))
-          .map(v -> Float.parseFloat(v)).collect(Collectors.toList());
+      List<Float> floats;
+      try {
+        floats = MAPPER.readValue(value, TYPE_REFERENCE);
+      } catch (Exception e) {
+        log.warn("String: {} cannot be parsed.", value, e);
+        return null;
+      }
       doc.removeField(this.field);
```

Comment:
would you please make a ticket to update this stage to use a "source" and "dest" approach like other stages.
It seems overly limit that this stage only supports overwriting the source field.
This work can be done later, in a separate PR.